src/containers/Home.jsx

```diff
@@ -87,10 +87,10 @@ const Content = (props) => {
       <Row>
         <Col sm={7} className="mx-3 mx-sm-0 blurb-content">
           <div>
-            <h3>Curious about life in SE? Check out our latest feature: </h3>
+            <h3>Want to stay connected with the SE community? Join our Discord: </h3>
```

Comment:
For better formatting, maybe we can say "Want to stay connected?"
and then for the link text say "Join our UW Software Engineering Discord" instead of "Click this invite link"